variables.tf

```diff
@@ -235,3 +241,11 @@ variable "node_to_node_encryption_enabled" {
   default     = false
   description = "Whether to enable node-to-node encryption"
 }
+
+locals {
```

Comment:
Please, move `locals` to main.tf

Reply:
I have described issue here https://github.com/cloudposse/terraform-aws-elasticsearch/issues/57